### DISCUSSION TITLE
Working with cookies

### DISCUSSION THREAD

**Discussion by Facj** [codesandbox/codesandbox-client#6347] [Q&A]
Is it possible to use cookies in sandbox? I have been unable to make it work.
I have tried a simple example that works fine on my PC to set one cookie.
Sandbox [here](https://codesandbox.io/s/protected-route-i356j).

I also would like to know if it is possible to see the cookies stored in the browser provided by the sandbox, the same way it is done by doing inspection on regular browsers.

**Comment by JamesACS** [comment]
Hey @Facj , 

Due to iFrame security this won't work in the preview window within the CodeSandbox interface - however if you pop the preview out into a new window it will be able to read and set cookies
